run_sets/testsuite.yml

```diff
@@ -43,6 +43,8 @@
 - features/salt.feature
 - features/salt_minions_page.feature
 - features/salt_minion_details.feature
+- features/salt_download_endpoint.feature
```

Comment:
Exactly. You can change the execution order if you want, they would pass as they're now idempotent 😎